src/mail.rs

```diff
@@ -126,21 +191,27 @@ impl<'a> Mail<'a> {
         Ok(self)
     }
 
-    /// Add content for inline images in the message.
-    add_field!(add_content <- content: &'a str);
+    add_field!(
+        /// Add content for inline images in the message.
+        add_content <- content: &'a str
+    );
 
-    /// Add a custom header for the message. These are usually prefixed with
-    /// 'X' or 'x' per the RFC specifications.
-    add_field!(add_header <- headers: &'a str);
+    add_field!(
+        /// Add a custom header for the message. These are usually prefixed with
+        /// 'X' or 'x' per the RFC specifications.
+        add_header <- headers: &'a str
+    );
 
     /// Used internally for string encoding. Not needed for message building.
     pub(crate) fn make_header_string(&mut self) -> SendgridResult<String> {
         let string = serde_json::to_string(&self.headers)?;
         Ok(string)
     }
 
-    /// Add an X-SMTPAPI string to the message. This can be done by using the
-    /// 'rustc_serialize' crate and JSON encoding a map or custom struct. Or
-    /// a regular String type can be escaped and used.
-    add_field!(add_x_smtpapi = x_smtpapi: &'a str);
+    add_field!(
+        /// Add an X-SMTPAPI string to the message. This can be done by using the
+        /// 'rustc_serialize' crate and JSON encoding a map or custom struct. Or
```

Comment:
The rustc_serialize crate is deprecated so it may be good to advocate for serde here.

Reply:
Yeah, I assume this was a leftover from the early days of the crate?